shared/src/commonMain/kotlin/com/learndiscussvote/projectguardian/shared/PoliticalRaceModel.kt

```diff
@@ -226,11 +235,29 @@ data class PoliticalEntityModel internal constructor(
+                     RawPoliticalStatementModel(statement = Sourced(data = "We support lower and fair tax models that adequeately fund government without undue waste or programs better facilitated by the private sector.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.TAXES_FUNDING, listOf(PoliticalStatementCategory.GOVERNMENT OPERATION), 
+                     RawPoliticalStatementModel(statement = Sourced(data = "We condemn decisions by activist judges to allow the desecration of the flag and to deny children the opportunity to say the Pledge of Allegiance in public schools.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.PATRIOTISM, listOf(PoliticalStatementCategory.EDUCATION), 
+                     RawPoliticalStatementModel(statement = Sourced(data = "We support standing at the presentation of our national anthem.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.PATRIOTISM),
+                     RawPoliticalStatementModel(statement = Sourced(data = "We support traditional marriage and family as being the foundation for a free society.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.FAMILY),
+                     RawPoliticalStatementModel(statement = Sourced(data = "We support the appointment of justices and judges who respect the constitutional limits on their power and respect the authority of the states to decide such fundamental social questions.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.GOVERNMENT OPERATION),
+                     RawPoliticalStatementModel(statement = Sourced(data = "We support Alabama as a 'Right to Work' state.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.JOBS_ECONOMY),
+                     RawPoliticalStatementModel(statement = Sourced(data = "We oppose the implementation of the Common Core Curriculum.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.EDUCATION),
+                     RawPoliticalStatementModel(statement = Sourced(data = "We believe that business growth must be encouraged by all levels of government through elimination of all unnecessary regulations that unduly burden industry and businesses.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.JOBS_ECONOMY, listOf(PoliticalStatementCategory.GOVERNMENT OPERATION),
+                     RawPoliticalStatementModel(statement = Sourced(data = "We support legislation requiring proof of citizenship when registering to vote and secure photo identification when voting in order to help ensure that we have fair and honest elections.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.DEMOCRACY_VOTING, listOf(PoliticalStatementCategory.GOVERNMENT OPERATION),                           
```

Comment:
```suggestion
                     RawPoliticalStatementModel(statement = Sourced(data = "We support legislation requiring proof of citizenship when registering to vote and secure photo identification when voting in order to help ensure that we have fair and honest elections.", ArchivedSource.alabamaRepubicanParty), PoliticalStatementCategory.DEMOCRACY_VOTING, listOf(PoliticalStatementCategory.GOVERNMENT OPERATION)),                           
```

Have to have a matching sets of parentheses haha